content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -431,45 +480,44 @@ Some values of an object are typically generated before the object is persisted.
 
 ## Server Side Apply
 
-Starting from Kubernetes v1.18, you can enable the
-[Server Side Apply](/docs/reference/using-api/server-side-apply/)
-feature so that the control plane tracks managed fields for all newly created objects.
+Kubernetes' [Server Side Apply](/docs/reference/using-api/server-side-apply/)
+feature allows the control plane to tracks managed fields for newly created objects.
 Server Side Apply provides a clear pattern for managing field conflicts,
 offers server-side `Apply` and `Update` operations, and replaces the
-client-side functionality of `kubectl apply`. For more details about this
-feature, see the section on
-[Server Side Apply](/docs/reference/using-api/server-side-apply/).
+client-side functionality of `kubectl apply`.
 
-## Resource Versions
+See [Server Side Apply](/docs/reference/using-api/server-side-apply/) for more details.
 
-Resource versions are strings that identify the server's internal version of an object. Resource versions can be used by clients to determine when objects have changed, or to express data consistency requirements when getting, listing and watching resources. Resource versions must be treated as opaque by clients and passed unmodified back to the server. For example, clients must not assume resource versions are numeric, and may only compare two resource versions for equality (i.e. must not compare resource versions for greater-than or less-than relationships).
+## Resource versions
 
-### ResourceVersion in metadata
+Resource versions are strings that identify the server's internal version of an object. Resource versions can be used by clients to determine when objects have changed, or to express data consistency requirements when getting, listing and watching resources. Resource versions must be treated as opaque by clients and passed unmodified back to the server. For example, clients must not assume resource versions are numeric, and may only compare two resource versions for equality (this means that you must not compare resource versions for greater-than or less-than relationships).
+
+### `ResourceVersion` in metadata {#resourceversion-in-metadata}
 
 Clients find resource versions in resources, including the resources in watch events, and list responses returned from the server:
 
 [v1.meta/ObjectMeta](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#objectmeta-v1-meta) - The `metadata.resourceVersion` of a resource instance identifies the resource version the instance was last modified at.
 
 [v1.meta/ListMeta](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#listmeta-v1-meta) - The `metadata.resourceVersion` of a resource collection (i.e. a list response) identifies the resource version at which the list response was constructed.
 
-### The ResourceVersion Parameter
+### `ResourceVersion` parameter {#the-resourceversion-parameter}
```

Comment:
Should the `resourceVersion` parameter in a heading list as `ResourceVersion` parameter?

Reply:
Addressed, I think.